doc/COMPILE.md

```diff
@@ -502,6 +502,12 @@ This value specifies the root directory of a local copy of the [Google Test][] f
   top level directory of `GTEST_ROOT`. If this file exists, then the build system
   will use the sources files at `GTEST_ROOT` to translate tests that use [Google Test][].
 
+We currently pin Google Test to commit
```

Comment:
This information is not relevant here. It should be in BUILDSERVER or, if you insist, TESTING (its only partly true due to maintainer builds).

What belongs here is that GTEST_ROOT can be an environment variable (if we really want that behavior).

Reply:
I added it here as I see it as a source of potential testing / compilation differences between the test system + users.

That GTEST_ROOT defaults to the value of a environment variable should not be needed as it can still be set via -DGTEST_ROOT and that always overrides the value pulled form ENV.

Reply:
Yes, it is a good idea but it is also a goal to make Elektra easy to compile+install. And requiring to download and install a specific version does not make things easy. The most important info for devs is that it will be downloaded anyway and the most important info for maintainer is how to use GTEST_ROOT.

To have reproducibility in TESTING we can describe this in TESTING.md but more importantly: devs should have an easy way to get your excellent Docker images which improves reproducibility much more than using the same gtest version.